router/http.go

```diff
@@ -238,7 +238,23 @@ func (h *httpSyncHandler) Set(data *router.Route) error {
 	service.refs++
 	r.service = service
 	h.l.routes[data.ID] = r
-	h.l.domains[strings.ToLower(r.Domain)] = r
+	if data.Path == "/" {
+		log.Printf("router: adding default route for domain: %s", r.Domain)
```

Comment:
Use `log15` (there is a global [logger variable](https://github.com/flynn/flynn/blob/master/router/server.go#L20))